### DISCUSSION TITLE
What is "Healthy"?

### DISCUSSION THREAD

**Discussion by jetkins** [alexjustesen/speedtest-tracker#1934] [Q&A] [answered]
Poking around in the UI, I discovered the optional Healthy column on the Results page, but I can't find any mention of it anywhere in the docs.  The column is blank for every single result, regardless of whether it was successful or failed.  I can filter on it too, but apparently everty single result is "unhealthy".  Am I missing something, or did this just get abandoned but never removed from the code?

**Comment by jetkins** [comment]
I did find one [PR](https://github.com/alexjustesen/speedtest-tracker/pull/1814) that mentions it, but it refers to benchmark results, which also appears to be a dead end.

**Comment by jetkins** [comment]
If it's intended to flag results that don't meet the thresholds I've set, then it's not working - I get alerts when my thresholds are breached, but those results are not flagged in any way.  Or perhaps the results that _do_ meet thresholds should be getting marked at Healthy, but that's not happening.

**Reply by svenvg93** [reply, marked as answer]
It does indeed refers to the threshold. Whenever a threshold is breaded `healthy` is set to `false`. 

It does work for me. 
<img width="1250" alt="Scherm­afbeelding 2024-12-17 om 19 36 46" src="https://github.com/user-attachments/assets/63654d45-e7f2-4c4e-87fa-6085a5f4657a" />

The only thing that I just noticed is that filtering on them does not work. Made #1935 for it

**Comment by jetkins** [comment]
Thanks for responding.  Weirdly, now that I'm back at home, I see the results flagged appropriately as Healthy or Unhealthy, though I swear I wasn't seeing that from my PC at work.  Maybe I was inadvertently looking at results from before I set my thresholds, maybe I'm just getting delusional - I'll have to check again tomorrow.

**Reply by svenvg93** [reply]
No worries, results from before introducing this won't have this shown indeed.
